content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,148 @@
 
 ## Security
 
-- Deprecated `RbacConfig` replacing it with `ClusterRbacConfig` to implement the correct cluster scope.
-  Refer to our guide on [Migrating the `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/#migrating-the-rbacconfig-to-clusterrbacconfig)
-  for migration instructions.
+- **Readiness and Liveness Probes**. Added support for Kubernetes' HTTP [readiness and liveness probes when mutual TLS is enabled](/help/faq/security/#k8s-health-checks).
+
+- **Cluster RBAC Configuration**.  Replaced the `RbacConfig` resource with the `ClusterRbacConfig` resource to implement the correct cluster scope.
+See [Migrating `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrading-istio#migrating-the-rbacconfig-to-clusterrbacconfig)
+for migration instructions.
+
+- **Identity Provisioning Through SDS**. Provides stronger security with on-node key generation and dynamic certificate rotation without restarting Envoy.
+See [Provisioning Identity through SDS](/docs/tasks/security/auth-sds) for more information.
+
+- **Authorization for TCP Services**. Supports authorization for TCP services in addition to HTTP and gRPC services. 
+See [Authorization for TCP Services](/docs/tasks/security/authz-tcp) for more information.
+
+- **Authorization for End-User Groups**. Allows authorization based on `groups` claim or any list-typed claims in JWT.
+See [Authorization for groups and list claims](/docs/tasks/security/rbac-groups/) for more information.
+
+- **End-User Authentication with Per-Path Requirements**. Allows you to enable or disable JWT authentication based on the request path.
+See [End-user authentication with per-path requirements](/docs/tasks/security/authn-policy/#end-user-authentication-with-per-path-requirements) for 
+more information.
+
+- **External Certificate Management on Ingress Gateway Controller**. Dynamically loads and rotates external certificates.
+
+- **Vault PKI Integration**. Provides stronger security with Vault-protected signing keys and facilitates integration with existing Vault PKIs.
+See [Istio Vault CA Integration](/docs/tasks/security/vault-ca) for more information.
+
+- **Customized (non `cluster.local`) Trust Domains**. Supports organization- or cluster-specific trust domains in the identities.
+
+- TBD: How about adding istio/istio#11667 as well? As this is also a significant feature for security which can enable end user to set
+different CA and Certs for different namespaces.
+
+## Multiclusters
+
+- **Non-Routable L3 Networks**. Enabled using a single Istio control plane in multicluster environments with non-routable
+L3 networks.
+TBD: LINK
+
+- **Multiple Control Planes**. Added support for multiple Istio control planes in support of multicluster environments.
+TBD: LINK
+
+## Policies and telemetry
+
+- **Policy Checks Off By Default**. Changed policy checks to be turned off by default which improves performance for most customer scenarios.
```

Comment:
I feel like this deserves some sort of special warning / call out.  @mandarjog have you written a doc on how to enable policy checking for the website yet?  If not, we need one (imho).

Reply:
Yes, we need to call out any breaking changes right at the top explicitly.

Reply:
Filed to track docs changes: https://github.com/istio/istio.io/issues/3671